mountinfo/mountinfo_linux.go

```diff
@@ -18,11 +18,9 @@ import (
 func GetMountsFromReader(r io.Reader, filter FilterFunc) ([]*Info, error) {
```

Comment:
Can we have a UT?

Reply:
Added. There were a few for this function already, just not one for the `FilterFunc` callback.